lib/core/src/Cardano/Wallet/Primitive/Types/Address.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE DeriveGeneric #-}
+
+-- |
+-- Copyright: © 2018-2020 IOHK
+-- License: Apache-2.0
+--
+-- This module provides the 'Address' data type, which provides a human-friendly
```

Comment:
How about `This module provides the main 'Address' data type used by the wallet`, or something to more show the relationship this module has with the rest of the code-base?

Reply:
> How about This module provides the main 'Address' data type used by the wallet, or something to more show the relationship this module has with the rest of the code-base?

Very good point! I'll add this in a follow-up PR, along with the other suggestions.